pkg/resources/view.go

```diff
@@ -24,20 +24,20 @@ var viewSchema = map[string]*schema.Schema{
 	"name": {
 		Type:             schema.TypeString,
 		Required:         true,
-		Description:      "Specifies the identifier for the view; must be unique for the schema in which the view is created. Don't use the | character.",
+		Description:      blocklistedCharactersFieldDescription("Specifies the identifier for the view; must be unique for the schema in which the view is created."),
 		DiffSuppressFunc: suppressIdentifierQuoting,
 	},
 	"database": {
 		Type:             schema.TypeString,
 		Required:         true,
-		Description:      "The database in which to create the view. Don't use the | character.",
+		Description:      "The database in which to create the view.",
```

Comment:
Use blocklisted characters function here and in the schema as well (can be done in the next pr).

Reply:
Done.